incubator/clickhouse/templates/statefulset.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: apps/v1beta1
```

Comment:
This can be incremented to `apps/v1`. All version of k8s supported by the k8s community have this version.

Reply:
Yes, I ran it old cluster, but it is now fixed